integration/client-cli/client-registration-cli/README-windows.md

```diff
@@ -0,0 +1,197 @@
+Keycloak Client Registration CLI
```

Comment:
Please remove this file. We don't want documentation in multiple places and this file will just end up being out of date.